EXAMPLES.md

```diff
@@ -188,6 +188,27 @@ await client.loginWithPopup({
 });
 ```
 
+### Switch to a different organizations
+
+When working with multiple organizations, there might be a situation where you want your users to be able to switch between different organizations.
+
+Important to realize is that the authentication state in the SDK is build around the organization, so in order to switch organizations, you need to logout and login again. However, it's enough to log out from the SDK and not from Auth0. Doing that ensures that, when calling login again, the user might not be prompted for their credentials depending on whether or not they have an active session with Auth0.
```

Comment:
How about? 

"To do this, clear the local logged in state from your application and login to Auth0 again, leveraging any existing Auth0 session to prevent the user from being prompted for their credentials."